.changeset/curly-zoos-worry.md

```diff
@@ -0,0 +1,11 @@
+---
+"saleor-app-data-importer": minor
+"saleor-app-products-feed": minor
+"saleor-app-monitoring": minor
+"saleor-app-invoices": minor
+"saleor-app-klaviyo": minor
+"saleor-app-search": minor
+"saleor-app-slack": minor
+---
+
+VercelAPL can no longer be set for the app, since its deprecated and will be removed in app-sdk 0.30.0
```

Comment:
```suggestion
VercelAPL can no longer be set for the app since it's deprecated and will be removed in app-sdk 0.30.0
```
Isn't that a breaking change? Seems to be a major-worthy.

Reply:
I dont know, maybe? Im not sure if we treat apps as "production" at the moment. Entire SDK is pre-prod

I can bump them to 2.x.x but how do we version AppManifest then? App will use the same manifest but it will be 2.0.0?

Reply:
I agree with this reasoning :+1: 
We'll need to figure this out during "production" milestone